external/harvestPackages/harvestPackages.props

```diff
@@ -91,6 +91,12 @@
 <PackageReference Include="System.Runtime.CompilerServices.Unsafe">
   <Version>4.5.1</Version>
 </PackageReference>
+<PackageReference Include="System.Runtime.WindowsRuntime">
+  <Version>4.3.0</Version>
+</PackageReference>
+<PackageReference Include="System.Runtime.WindowsRuntime">
```

Comment:
Did you mean for this one to be .UI.Xaml?

Reply:
Yes...